PublicFolders/SourceSideValidations/Tests/Permission/Remove-InvalidPermission.ps1

```diff
@@ -51,6 +57,16 @@ function Remove-InvalidPermission {
                             $perm | Remove-PublicFolderClientPermission -Confirm:$false
                         }
                     }
+
+                    if (
+                        ($null -ne $permission.User.ADRecipient) -and
+                        ($permission.User.ADRecipient.RecipientDisplayType.ToString() -notin $validACLableRecipientTypes)
+                    ) {
+                        if ($PSCmdlet.ShouldProcess("$($result.FolderIdentity)", "Remove $($perm.User.DisplayName)")) {
+                            Write-Host "Removing $($perm.User.DisplayName) from folder $($result.FolderIdentity)"
+                            $perm | Remove-PublicFolderClientPermission -Confirm:$false
```

Comment:
@bill-long Because we are adding a change to a "Write" type here, we need to send this over to the test team before merging too main. 

Might be best to place both this PR and the other PR that contains a change to Write to a single branch to have the test team test out that version.

Reply:
Let's just build out of the latest PR. The testing of that can apply to both.